content/collections/docs/conditional-fields.md

```diff
@@ -16,6 +16,14 @@ Field conditions are set on individual field settings in [blueprints](/blueprint
 
 You may specify various rules for showing a field under either the `if` / `show_when` keys, or hiding a field under the `unless` / `hide_when` keys.
 
+### Data Flow
+
+Only visible fields are submitted with your form data by default. This allows you to control data flow, and [conditionally apply validation](#validation) to visible fields when needed.
```

Comment:
It says `by default` but for now it's the only way.

Reply:
True. That was in prep for the upcoming config, but I'll remove that for now.